cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/declarations/FunctionDeclaration.kt

```diff
@@ -106,12 +106,8 @@ open class FunctionDeclaration : ValueDeclaration(), DeclarationHolder, EOGStart
                     }
                 })
 
-    /**
-     * This function checks, if the supplied [CallExpression] has the same signature as the current
-     * [FunctionDeclaration].
-     */
-    fun hasSignature(call: CallExpression): Boolean {
-        return hasSignature(call.signature, call.arguments)
+    fun hasSignatureWithImplicitCast(call: CallExpression): Boolean {
```

Comment:
Necessary/Ever used? Remove from this PR or implement it ;)

Reply:
Not used. can be removed